### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by jordanruthe** [KlipperScreen/KlipperScreen#35] [General]
The general roadmap for features right now is as follows:

* Refine bed level panel to include more/less than 4 screws. 
* Refine bed mesh manager panel including plotting
* Folders for the print panel
* Update Manager

If you have any other ideas for features you would like to see, please contribute them in this thread.

**Comment by symekutz** [comment]
Have you put any thought into what it would take to make KlipperScreen able to switch between printers if multiple Klipper instances are running on the same pi?

**Reply by jordanruthe** [reply]
Sorry for the delay, but yes. There is currently another branch, mulitple_printers, that is starting to implement this functionality.

**Comment by rjmessing** [comment]
Would it be feasible to have the set point value displayed constantly?  I see the Temp/Setpoint for when the temperature is set but if I set 0 to turn the heaters off I don't see any value and am not sure if the heater is off or on if the temperature was at a stable temp, till it decides to fall.

**Comment by AlxSZP** [comment]
Hello! I'd like to submit a feature request. Recently OctoDash on their latest release enabled the possibility to turn on the printer when the screen is woken up. This supports TP Link, Tasmotta and Wemo devices
Any way this can be done as well?

**Reply by jordanruthe** [reply]
If you name your power device 'printer', you can turn it on from the splash screen (when the printer is off)

**Comment by MakManIfest** [comment]
Would be possible to make the extrude distance and extrude speed values for each button in the extrude panel user definable? Perhaps through the Klipperscreen settings menu?

**Comment by alfrix** [comment]
everything in here has been done
